backend/wmg/data/ontology_labels.py

```diff
@@ -22,10 +22,10 @@
 
 def ontology_term_label(ontology_term_id: str) -> Optional[str]:
     """
-    Returns the label for an ontology term, given its id. This excludes gene ontology term, which are handled
-    separately by gene_gene_term_label(). Return None if ontology term id is invalid.
+    Returns the label for an ontology term, given its id. This excludes gene ontology term
+    and self reported ethnicity term, which are handled separately by gene_term_label()
+    and ethnicity_term_label() respectively. Return None if ontology term id is invalid.
     """
-    global ontology_term_id_labels
```

Comment:
@atarashansky - Why do you say that? Python operates by _lexical scoping_ which means that it will look in its parent frame for `ontology_term_id_labels` which is the `ontology_lables.py` file where, upon import, declares `ontology_term_id_labels = {}` and `gene_term_id_labels = {}` in [lines 115-116](https://github.com/chanzuckerberg/single-cell-data-portal/blob/32b8b0edbdb75fb065d9b72ed8b7c4ceadd7037d/backend/wmg/data/ontology_labels.py#L115)

I think `global` is only required when you want to mutate a variable that is in an ancestor frame as in [__load_ontologies() function](https://github.com/chanzuckerberg/single-cell-data-portal/blob/32b8b0edbdb75fb065d9b72ed8b7c4ceadd7037d/backend/wmg/data/ontology_labels.py#L89) and [__load_genes() function](https://github.com/chanzuckerberg/single-cell-data-portal/blob/32b8b0edbdb75fb065d9b72ed8b7c4ceadd7037d/backend/wmg/data/ontology_labels.py#L98C8-L98C8)

In the case of mutation in functions like `__load_ontologies()`, if you didn't declare `global ontologies_term_id_labels` you would get an `UnboundLocalError`.

More info here: https://realpython.com/python-use-global-variable-in-function/#the-global-keyword

Moreover, passing tests and a functioning rdev indicate that this refactor is working

Reply:
Oh, lol, I didn't see lines 115-116 (which is funny considering that I was the one who added it there). I think the "global" keywords are vestigial, then...

Reply:
Thanks for the clarification!